src/api/configureApi.ts

```diff
@@ -167,7 +167,8 @@ export function configureAndStartApi(useMetrics: boolean = true) {
 
   app.use(getErrorLoggingMiddleware());
   const errorHandler: ErrorRequestHandler = (err, req, res, _next) => {
-    logger.error(`Fallback error handler invoked: ${err.message}`);
+    const traceId = req?.headers?.traceId?.toString() ?? '';
```

Comment:
`req.headers.traceId` has the type `string | string[] | undefined` for some reason, hence the need for `.toString()`